include/VerificationValidationWidget.h

```diff
@@ -15,6 +15,7 @@
 #include <QHBoxWidget.h>
 #include "VerificationValidation.h"
 #include "Utils.h"
+#include "map"
```

Comment:
i was able to compile without this header; there doesn't seem to be a local "map" header file either